app/src/main/res/layout/activity_new_profile.xml

```diff
@@ -12,14 +12,15 @@
         android:id="@+id/profile_sign_out"
```

Comment:
On the NewProfileActivity the friend's button is invisible if the user is looking at his profile. The only way to see his friends will be through the rankings if I'm not mistaking, should we also add the possibility to display a list of friends accessible from the user profile?

Reply:
I think we decided to not have it there as we want all the social part to be in the social activity.